frontend/vite.config.ts

```diff
@@ -11,40 +22,17 @@ export default defineConfig({
                 ? [reactRefresh()]
                 : []
         ),
+        dynamicApp(app),
     ],
     build: {
         rollupOptions: {
-            input: {
-                maker: resolve(__dirname, "maker.html"),
-                taker: resolve(__dirname, "taker.html"),
-            },
+            input: resolve(__dirname, `index.html`),
         },
+        outDir: `dist/${app}`,
     },
     server: {
```

Comment:
```suggestion
    server: {
      open: "/trade",
```

Reply:
if you don't mind, the `open` call was nice as it gave us the same behavior as we had before introducing vite (open the browser when starting the app). 
It could be either `"/" or "/trade" (I'd propose the latter, as we don't forward to `/trade` by default - maybe we could?)

Reply:
I will do `/` to encourage you to fix the app such that it also has a convenient homepage when the user is going to see it :)

Reply:
fair enough :)